content/SCALE/SCALETutorials/DataProtection/CloudSyncTasks/AddGooglePhotosCloudSyncTask.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Adding a Google Photos Cloud Sync Task"
+description: "This article provides instructions on how to set up Google Photos API credentials and use them to create a cloud sync task."
+weight: 40
+tags:
+- scalecloud
+- scalesgooglephotos
+- scalebackup
+---
+
+{{< toc >}}
+
+Google Photos works best in TrueNAS using a Google Photos API key and [rclone](https://rclone.org/) token.
+
+## Creating the API Credentials
+
+On the [Google API dashboard](https://console.cloud.google.com/apis/dashboard), click the dropdown menu next to the Google Cloud logo and select your project. If you do not have a project, click **NEW PROJECT** and enter a **Project name**, **Organization**, and **Location**.
+
+![GooglePhotosAPINewProject](/images/SCALE/22.12/GooglePhotosAPINewProject.png "New Project")
+
+### Enable API
+
+After you select your project, click **Enabled APIs & Services** on the left menu, then click **+ ENABLE APIS AND SERVICES**.
+
+![GooglePhotosAPIEnableAPIsandServices](/images/SCALE/22.12/GooglePhotosAPIEnableAPIsandServices.png "Enable APIs and Services")
+
+Enter *photos library api* in the search bar, then select **Photos Library API** and click **ENABLE**.
+
+![GooglePhotosAPIPhotosLibraryAPISearch](/images/SCALE/22.12/GooglePhotosAPIPhotosLibraryAPISearch.png "Photos Library API Search")
+
+![GooglePhotosAPIClickEnable](/images/SCALE/22.12/GooglePhotosAPIClickEnable.png "Click Enable")
+
+### Configure Authentication
+
+Next, click **OAuth consent screen** on the left menu, select **EXTERNAL**, then click **CREATE**.
+
+![GooglePhotosAPICreateExternal](/images/SCALE/22.12/GooglePhotosAPICreateExternal.png "Create External")
+
+Enter an **App name** and **User support email**. 
```

Comment:
Same reason as above. It is best to not use the field name as the "value" to enter.
```suggestion
Enter a value in **App name** and **User support email**. 
```